queries.yaml

```diff
@@ -104,7 +118,6 @@ layers:
     area-inclusion-threshold: 1
   buildings:
     template: buildings.jinja2
-    start_zoom: 13
```

Comment:
In this particular case, per-feature `min_zoom`. The `source`s themselves now have a start zoom for when querying should begin, but there are now cases like this one where we are relying more on the min zoom for filtering.